portfolio/src/main/client/res/style/pages/index.scss

```diff
@@ -0,0 +1,64 @@
+@use '../responsive';
+
+#index-hero {
+  margin-bottom: -8rem;
+  background-image: url('~@res/img/me/lake.jpg');
+  // position at 30% so my face is not cut off
+  background-position: center 30%;
```

Comment:
Have you tested this on different screen sizes? Want to make sure it looks good on the user's monitor too

Reply:
Yep